contracts/tokens/SecurityToken.sol

```diff
@@ -294,27 +286,27 @@ contract SecurityToken is ERC20, ERC20Detailed, ReentrancyGuard, RegistryUpdater
     * @param _module address of module to unarchive
     */
     function removeModule(address _module) external onlyOwner {
```

Comment:
Assume this is due to contract size issues?

Reply:
Oh yes. I totally forgot about this. I also commented out tests that required removal of modules.

Do you prefer to merge this as it as and reduce the size of ST in future or reduce the size before merging?

Reply:
Let's merge and create an issue to uncomment the function as a reminder.